portal-frontend/src/app/features/applications/edit-submission/primary-contact/primary-contact.component.html

```diff
@@ -1,67 +1,96 @@
-    <ul>
-      <li>Authorization Letters (if applicable)</li>
-    </ul>
-  </div>
 </div>
-<section>
-  <div class="contacts">
-    <ng-container *ngFor="let owner of parcelOwners">
-      <h5>{{ owner.displayName }}</h5>
-      <button
-        (click)="onSelectOwner(owner.uuid)"
-        *ngIf="owner.uuid !== selectedOwnerUuid"
-        color="primary"
-        [ngClass]="{ 'mat-error': !selectedOwnerUuid && showErrors }"
-        mat-stroked-button
-      >
-        Make Primary Contact
-      </button>
-      <div class="selected" *ngIf="owner.uuid === selectedOwnerUuid"><mat-icon>check</mat-icon>Primary Contact</div>
-    </ng-container>
-    <ng-container *ngIf="isGovernmentUser || selectedLocalGovernment">
-      <h5>{{ governmentName ?? 'Local / First Nation Government' }} Staff</h5>
-      <button
-        (click)="onSelectGovernment()"
-        *ngIf="!selectedLocalGovernment"
-        color="primary"
-        [ngClass]="{ 'mat-error': !selectedOwnerUuid && showErrors }"
-        mat-stroked-button
-      >
-        Make Primary Contact
-      </button>
-      <div class="selected" *ngIf="selectedLocalGovernment"><mat-icon>check</mat-icon>Primary Contact</div>
-    </ng-container>
-    <h5>Third-Party Agent</h5>
-    <button
-      *ngIf="!selectedThirdPartyAgent"
-      color="primary"
-      (click)="onSelectAgent()"
-      [ngClass]="{ 'mat-error': !selectedOwnerUuid && showErrors }"
-      mat-stroked-button
-    >
-      Make Primary Contact
-    </button>
-    <div class="selected" *ngIf="selectedThirdPartyAgent"><mat-icon>check</mat-icon>Primary Contact</div>
-    <div><!--this is required for correct responsive view --></div>
-    <app-validation-error *ngIf="!selectedOwnerUuid && showErrors">
-      Please select a primary contact
-    </app-validation-error>
-  </div>
 
+<section>
   <div class="agent-form">
     <h4>Primary Contact Information</h4>
     <form [formGroup]="form">
-      <div class="form-row">
+      <div class="full-row">
+        <p *ngIf="isGovernmentUser">
+          <strong>Will {{ governmentName ?? 'Local / First Nation Government' }} staff be the primary contact?</strong>
+        </p>
+        <p *ngIf="!isGovernmentUser">
+          <strong
+            >Will one of the landowners
+            <ng-container *ngIf="hasCrownParcels">or government contacts</ng-container> added previously be the primary
+            contact?</strong
+          >
+        </p>
+        <mat-button-toggle-group
+          class="input"
+          [formControl]="isExistingOwner"
+          (change)="onSelectPrimaryContactType(isExistingOwner.value)"
+        >
+          <mat-button-toggle [value]="true">Yes</mat-button-toggle>
+          <mat-button-toggle [value]="false">No</mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+
+      <!-- Owner -->
+      <div *ngIf="selectedThirdPartyAgent === false && parcelOwners.length > 0">
+        <label *ngIf="owners.length"
```

Comment:
Should this be parcelOwners.length rather than owners.length since the ngFor bellow refers to parcelOwners?

Reply:
Yes! Good catch. This was initially copied from step 1 where we want all owners.